### ISSUE TITLE
Error : missing module eslint-webpack-plugin on fresh install

### ISSUE THREAD

**Issue by SimonVanacco** [symfony/webpack-encore-bundle#163]
Hi ! 

I'm having this error on a fresh symfony install : 

![image](https://user-images.githubusercontent.com/5305627/150528073-02e8ee61-6daf-4cf8-9c0c-5e43106e90f6.png)

Steps to reproduce : 
```
 composer create-project symfony/skeleton my_project_directory 
 cd my_project_directory
 composer require webapp
 yarn install
 yarn encore prod
 # Error
```

Versions:
```
Ubuntu 20.04
Node 16.13.2
Yarn 1.22.10
```

Thanks a lot !

**Comment by Kocal** [comment]
Hi, this is not related to webpack-encore-bundle but to Encore itself.

However, this issue has been fixed this morning and a new version ([1.8.1](https://github.com/symfony/webpack-encore/releases/tag/v1.8.1)) has been released 30 minutes ago. 

So you need to upgrade `@symfony/webpack-encore` to 1.8.1 and it should fix your issue.

**Comment by SimonVanacco** [comment]
Got confused about where to report sorry ! Thanks !
